src/widget/auth/book.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback, useEffect, useState } from "react";
+
+import { motion, useAnimate } from "framer-motion";
+
+type Props = {
+  children: React.ReactNode;
+  isTriggered: boolean;
+  onComplete?(): void;
+};
+
+export const Book = ({ children, isTriggered, onComplete }: Props) => {
+  const duration = 2;
+
+  const onStart = useCallback(() => {
+    // может, придется добавить глобальный класс для body, пока неизвестно
+    document.body.classList.add("no-scrollbar");
+  }, []);
+
+  const onEnd = useCallback(() => {
+    document.body.classList.remove("no-scrollbar");
+    onComplete?.();
+  }, [onComplete]);
+
+  const [zIndex, setZIndex] = useState(2);
+  const [scope, animate] = useAnimate();
+
+  const run = useCallback(() => {
+    animate(
+      scope.current,
+      { translateX: "50%" },
+      {
+        duration: 2,
+        onComplete: onEnd,
+        onPlay: onStart,
+        onUpdate: (x) => {
+          if (x > 10) setZIndex(0);
+        },
+      }
+    );
+  }, [onEnd, onStart, animate, scope]);
+
+  useEffect(() => {
+    if (isTriggered) {
+      run();
+    }
+  }, [isTriggered, run]);
```

Comment:
Let's remove this useEffect
```suggestion
  useAccountEffect({
    onConnect: () => {
      run();
    },
  });
```